docs/api.md

```diff
@@ -558,7 +560,8 @@ will not be processed). The server will respond with a fake response that your
 client **MUST NOT** process or parse. The response will not be a valid JSON
 object.
 
-Client's should sporadically issue chaff requests to mirror real-world usage.
+Client's should sporadically issue chaff requests to mirror real-world usage
+for both the `/verify`, `/certificate`, and key server stats endpoints.
```

Comment:
key server publish endpoints, not stats.

Reply:
Done